docs/DevGuide/Protocols.md

```diff
@@ -32,12 +37,23 @@ For example, the following class conforms to the protocol we just defined:
 
 \snippet Utilities/Test_ProtocolHelpers.cpp named_conformance
 
+The class indicates it conforms to the protocol by (publicly) deriving off
+`ConformsTo<TheProtocol>`.
+
 Once you have defined a protocol, you can check if a class conforms to it using
 the `conforms_to` metafunction:
 
 \snippet Utilities/Test_ProtocolHelpers.cpp conforms_to
 
-So now you can write code that relies on the interface defined by the protocol:
+Note that checking for protocol conformance is cheap, because the `conforms_to`
+metafunction only checks if the class _indicates_ it conforms to the protocol.
```

Comment:
it would be helpful to insert "_indicates_ it conforms to the protocol via the above inheritance" just to reiterate and make the statement totally clear.